### DISCUSSION TITLE
I would like to resubmit my academic affiliation documents.

### DISCUSSION THREAD

**Discussion by junghyungo** [community/community#137464] [GitHub Education]
### Hi everyone,

I submitted a document in Korean last month, and there's an issue with it being rejected.   
I'm trying to resubmit it as an English document, but I can't find the submit button.   

<img width="749" alt="Screenshot 2024-09-02 at 12 29 17 PM" src="https://github.com/user-attachments/assets/f3485c03-72b7-41b7-98de-b3477038dca9">

**Comment by alx-mp4** [comment]
Hi @junghyungo, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
